packages/docs/fluent-editor/demos/collaborative-editing.spec.ts

```
@@ -34,24 +45,62 @@ async function selectAll(page: Page): Promise<void> {
 
 let p1: Page, p2: Page, b1: Browser, b2: Browser
 
-test.beforeEach(async () => {
-  [p1, p2, b1, b2] = await openTwoPages()
+test.beforeAll(async ({ browser }) => {
+  const page = await browser.newPage()
+  try {
+    collabWsReachable = await page.evaluate((wsUrl) => new Promise<boolean>((resolve) => {
```

ESLint

[error] Unexpected parentheses around single function argument having a body with no curly braces.

(style/arrow-parens)